facade/service.go

```diff
@@ -1788,7 +1800,9 @@ func (f *Facade) WaitSingleService(svc *service.Service, dstate service.DesiredS
 
 func (f *Facade) StartService(ctx datastore.Context, request dao.ScheduleServiceRequest) (int, error) {
 	defer ctx.Metrics().Stop(ctx.Metrics().Start("Facade.StartService"))
-	return f.ScheduleServices(ctx, request.ServiceIDs, request.AutoLaunch, request.Synchronous, service.SVCRun, false)
+	alog := f.auditLogger.Action(audit.Start).Message(ctx, "Starting Service").Type(service.GetType()).ID(strings.Join(request.ServiceIDs, ", "))
```

Comment:
A couple of suggestions to emphasize that the user is starting one or more services:
* `"Starting Service(s)"`
* `WithField("ids", strings.Join(request.ServiceIDs, ", "))` instead if `ID(...)`
* Also add a field for `count` and log successCount